crates/primitives/src/lib.rs

```diff
@@ -78,16 +78,6 @@ pub use arbitrary;
 #[cfg(feature = "c-kzg")]
 pub use c_kzg as kzg;
 
-/// Optimism specific re-exports
-#[cfg(feature = "optimism")]
-mod optimism {
-    pub use crate::transaction::{optimism_deposit_tx_signature, TxDeposit, DEPOSIT_TX_TYPE_ID};
```

Comment:
should we remove these in the `mod.rs` as well?

Reply:
Done, was thinking the goal was to do in multiple PRs but this is cleaner like that, you're right. I just let

```rust
pub use tx_type::DEPOSIT_TX_TYPE_ID;
```

as is because `tx_type` is a private module.